users-have-many-projects

From: Bryan Larsen <bryan@larsen.st>

We also need the other end of this association, in the User model:

SHOW_PATCH
---
 app/models/user.rb |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/app/models/user.rb b/app/models/user.rb
index e406a43..6bf0d8b 100644
--- a/app/models/user.rb
+++ b/app/models/user.rb
@@ -14,6 +14,7 @@ class User < ActiveRecord::Base
 
   has_many :task_assignments, :dependent => :destroy, :inverse_of => :user
   has_many :tasks, :through => :task_assignments
+  has_many :projects, :class_name => "Project", :foreign_key => "owner_id", :inverse_of => :owner
 
   # This gives admin rights and an :active state to the first sign-up.
   # Just remove it if you don't want that
